packages/visx-zoom/src/types.ts

```diff
@@ -47,7 +47,7 @@ export interface ProvidedZoom {
   /** Callback for dragEnd, sets isDragging to false. */
   dragEnd: () => void;
   /** Callback for dragMove, results in a scale transform. */
-  dragMove: (event: React.MouseEvent | React.TouchEvent | MouseEvent | TouchEvent) => void;
+  dragMove: (event: React.MouseEvent | React.TouchEvent | MouseEvent | TouchEvent, options?: { offsetX?: number; offsetY?: number }) => void;
```

Comment:
lgtm but I think `prettier` is unhappy with this being on a single line. `yarn format` should work, but may also affect other files (fine to just not commit those, or feel free to!)

Reply:
Fixed :)